content(data tiering): expand (#1784)
timescale/docs: cloud/data-tiering/index.md

Timescale Cloud includes a low-cost object-storage layer built on Amazon S3.
This allows you to tier your hypertable data across different storage layers to
get the best price performance. You can use primary storage for data that

> requires quick access, and low-cost object storage for historical data. No
> matter where your data is stored, you can still query it with standard SQL.

Reviewer: "no matter" might not translate well.

```suggestion
requires quick access, and low-cost object storage for historical data. 
Regardless of where your data is stored, you can query it with standard SQL.
```

    move it to native object storage.

*   **Transparent SQL queries.** You can interact with your data normally even
    when it's distributed across different storage layers. Your hypertable is
    spread across the layers, so queries and `JOIN`s work and fetch the same
    data as usual. Reading data from object storage adds only a few tens of
    milliseconds in latency, which goes away for larger scans.

## Architecture

Data tiering works by periodically and asynchronously moving older chunks to S3

> storage. There, it's stored in a compressed columnar format well-suited for S3,
> namely, the Apache Parquet format. Data remains accessible both during and after

Reviewer: ```suggestion
storage. There, it's stored in the Apache Parquet format, which is a compressed 
columnar format well-suited for S3. Data remains accessible both during and after
```

migration.

> When you run your regular SQL queries, a behind-the-scenes process transparently

Reviewer: ```suggestion
When you run regular SQL queries, a behind-the-scenes process transparently
```
